src/ngraph/op/reverse.hpp

```diff
@@ -77,5 +77,59 @@ namespace ngraph
 
             AxisSet m_reversed_axes;
         };
+
+        namespace v1
+        {
+            class Reverse : public Op
+            {
+            public:
+                enum class Mode
+                {
+                    INDEX,
+                    MASK
+                };
+
+                NGRAPH_API
+                static const std::string type_name;
+                const std::string& description() const override { return type_name; }
```

Comment:
These need to be switched to `type_info`
See `abs.hpp`/`abs.cpp`
They should work as-is for a short time.